### ISSUE TITLE
HIP-RA & Monte Carlo user manual instructions

### ISSUE THREAD

**Issue by softwareengineerprogrammer** [NatLabRockies/GEOPHIRES-X#72]
1. [X] Monte Carlo: https://github.com/NREL/GEOPHIRES-X/pull/116
1. [X] HIP-RA: https://github.com/NatLabRockies/GEOPHIRES-X/pull/521

**Comment by softwareengineerprogrammer** [comment]
https://github.com/softwareengineerprogrammer/GEOPHIRES-X/commit/44b56fd527518ef4fd2cbcd26f3ecb80cc1aee35

**Comment by Bernard-Ngu** [comment]
I'd like to take this on. It looks like the Monte Carlo side is covered by docs/Monte-Carlo-User-Guide.md, but there's no equivalent for HIP-RA. I'd propose adding docs/HIP-RA-User-Guide.md following the same structure — overview, input parameters, a worked example from tests/examples/, and interpretation of outputs. Does that match what you had in mind for the remaining scope here?

**Comment by softwareengineerprogrammer** [comment]
@Bernard-Ngu yes, that is precisely correct! Feel free to ask any further questions here, or send a PR with your proposed change.
